package.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "@ni-private/ni-system-link-server-grafana-plugins",
+  "description": "Placeholder package for SystemLink Server plugins for Grafana",
```

sorry if wasn't clear, we should keep `name` and `version` (those two are the bare minimum required) and have the additional `description` and `private`

```suggestion
 "version": "1.0.0",
  "description": "Placeholder package for SystemLink Server plugins for Grafana",
```

---

no longer applies 👍

---

That was failure to read on my part, but thanks to your comment above the dummy package file is no longer required anyway.